lib/printer.js

```diff
@@ -1101,11 +1105,19 @@ function genericPrintNoParens(path, options, print) {
         return concat(parts);
 
     case "ClassProperty":
-        return concat([
-            path.call(print, "key"),
-            path.call(print, "typeAnnotation"),
-            ";"
-        ]);
+        var parts = [];
+        if (n.static)
+            parts.push("static ");
```

Comment:
I've started listening to my orthodontist and putting braces on everything, though you can be forgiven for following the existing style in this file :)